### PR TITLE
[Feature] Allow ordering of `/v1/accounts`

### PR DESCRIPTION
**Pull request by kilemensi**
## Description

Allow ordering of `/v1/accounts` for the UI to match designs.

## Type of change

- [x] New feature (non-breaking change which adds functionality)

## Screenshots

Screenshot from 2022-02-14 11-24-26

## Checklist:

- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code

### CODE REVIEW

File: twoopstracker/twoops/urls.py [hunk 1]
```diff
@@ -26,7 +26,7 @@
         "lists/upload", AccountsListUploadAPIView.as_view(), name="accounts_list_upload"
     ),
     path("lists/<pk>", AccountsList.as_view(), name="acccounts_list-detail"),
-    path("accounts/", TwitterAccountsView.as_view(), name="accounts"),
+    path("accounts/", TwitterAccountsView.as_view(), name="tweeter_account-list"),
```
Comment (esirK) [inline twoopstracker/twoops/urls.py:29]: Is `tweeter` the intended naming?

**Review by esirK — APPROVED** [review]
LGTM

File: twoopstracker/twoops/urls.py (hunk 1 above)
Comment (kilemensi) [reply twoopstracker/twoops/urls.py:29]: YOU KNOW it _ain't_!

Comment (kilemensi) [inline twoopstracker/twoops/urls.py:29]:
```suggestion
    path("accounts/", TwitterAccountsView.as_view(), name="twitter_account-list"),
```

**Pull request merged**